test/integration/ssl_integration_test.cc

```diff
@@ -17,15 +17,7 @@ using testing::Return;
 
 namespace Ssl {
 
-std::unique_ptr<Runtime::Loader> SslIntegrationTest::runtime_;
-std::unique_ptr<ContextManager> SslIntegrationTest::context_manager_;
-ServerContextPtr SslIntegrationTest::upstream_ssl_ctx_;
-ClientContextPtr SslIntegrationTest::client_ssl_ctx_plain_;
-ClientContextPtr SslIntegrationTest::client_ssl_ctx_alpn_;
-ClientContextPtr SslIntegrationTest::client_ssl_ctx_san_;
-ClientContextPtr SslIntegrationTest::client_ssl_ctx_alpn_san_;
-
-void SslIntegrationTest::SetUpTestCase() {
+void SslIntegrationTest::SetUp() {
```

Comment:
It's probably OK to keep it static but not ideal. It'll work as it's properly locked and we're not relying on any of the stats there AFAICT. Ideally each client would gets its own stats store, just need to keep track of the storage. You can just add a TODO to this effect if you'd like.